### DISCUSSION TITLE
mod or setting to ignore beatmap hitsound volume settings

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20784] [Ideas] [answered]
some badly designed maps set some important hitsound volumes to the minimum, making it impossible to tell whether you missed or its just the map. A mod or setting to set all hitsounds to 100% would fix this problem (maybe set reverse sliders to 40%)

(no I dont have specific examples, I was just remembering this and wanted to write it down before I forgot)

**Comment by ghost** [comment]
([banned forever](https://osu.ppy.sh/beatmapsets/16349#osu/64267) has some hitobjects marked with 0 volume. might record a video example later)

**Comment by peppy** [comment, marked as answer]
Please for the love of GOD search before spamming more discussions. I understand you may be trying to be helpful, but using the discussion entry field as a notepad for anything you think of is not too helpful. It's noisy and is not going to make anything happen faster. And on top of that, you're duplicating existing requests for the same thing more often than not.

https://github.com/ppy/osu/search?q=hit+sound+volume&type=discussions
